database/messaging.go

```diff
@@ -66,6 +68,26 @@ func (db *DB) RegisterScanListener(dbname, user, password, hostport, sslmode str
 
 	}()
 
+	go func() {
+		for {
+			_, err := db.Exec("update scans set ack=false where completion_perc=0 and timestamp < NOW() - INTERVAL '1 minute'")
+			if err != nil {
+				log.WithFields(logrus.Fields{
+					"error": err,
+				}).Error("Could not run zero completion update query")
+			}
+
+			_, err = db.Exec(fmt.Sprintf("select pg_notify('%s', ''||id ) from scans where ack=FALSE and timestamp < NOW() - INTERVAL '30 seconds'", listenerName))
```

Comment:
I think it would be wise to add a `LIMIT` to this query, because the first time we run it, it will notify ~4 million scans!!!
What about `LIMIT 1000`, which should be enough for 5 minutes of activity?